core/src/main/java/org/elasticsearch/plugins/PluginManagerCliParser.java

```diff
@@ -51,8 +49,26 @@
             .build();
 
     public static void main(String[] args) {
-        Environment env = InternalSettingsPreparer.prepareEnvironment(EMPTY, Terminal.DEFAULT);
-        LogConfigurator.configure(env.settings());
+        // initialize default for es.logger.level because we will not read the logging.yml
+        String loggerLevel = System.getProperty("es.logger.level");
+        if (loggerLevel == null) {
+            loggerLevel = "INFO";
+        }
+        // Set the appender for all potential log files to terminal so that other components that use the logger print out the
+        // same terminal.
+        // The reason for this is that the plugin cli cannot be configured with a file appender because when the plugin command is
+        // executed there is no way of knowing where the logfiles should be placed. For example, if elasticsearch
+        // is run as service then the logs should be at /var/log/elasticsearch but when started from the tar they should be at es.home/logs.
+        // Therefore we print to Terminal.
+        Environment env = InternalSettingsPreparer.prepareEnvironment(Settings.builder()
+                .put("appender.terminal.type", "terminal")
```

Comment:
added a very basic test that just checks the  6 lines are outputted (as before) if nothing was configured and more than 6 if we add -Des.logger.level=DEBUG. test is here: https://github.com/elastic/elasticsearch/pull/13933/files#diff-62e7a8fc15e3edc7b4bd8da86febbb40R356 Is that what you meant?